cmd/spire-agent/cli/run/run.go

```diff
@@ -381,6 +380,11 @@ func NewAgentConfig(c *Config, logOptions []log.Option) (*agent.Config, error) {
 	ac.Telemetry = c.Telemetry
 	ac.HealthChecks = c.HealthChecks
 
+	// TODO: remove deprecated configurable in 0.12.0
+	if c.Agent.DeprecatedEnableSDS != nil {
+		ac.Log.Warn("SDS support is now always on. The enable_sds configurable is ignored and should be removed.")
```

Comment:
The enable_sds configurable is now deprecated, and will be removed in a future release.

Slightly different suggested wording since this will be fatal as unknown config in the future. Maybe even be explicit about that? Want the log message to motivate people to update their config rather than just leave the ignored option set.

Reply:
Heh, I'm still willing to update the message if you have something you like better.